### PR TITLE
gc: add notes for tidb_gc_max_wait_time

### PR DESCRIPTION
**Pull request by TonsnakeLin**
Signed-off-by: TonsnakeLin <lpbgytong@163.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: garbage-collection-configuration.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ TiDB 5.0 及之后的版本不再需要向各个 TiKV Region 都发送触发 GC
 
 如果要了解 TiDB 历史版本中 GC 配置的变化信息，请使用左侧导航栏中的 _"TIDB 版本选择器"_ 切换到本文档的历史版本。
 
+## TiDB 6.1 引入的变化
```
Comment (Oreoxmt) [inline garbage-collection-configuration.md:34]:
```suggestion
## TiDB 6.1.0 引入的变化
```

File: garbage-collection-configuration.md [hunk 2]
```diff
@@ -31,6 +31,12 @@ TiDB 5.0 及之后的版本不再需要向各个 TiKV Region 都发送触发 GC
...
+
+在 TiDB 6.1 之前的版本中，TiDB 内部事务不会影响 GC Safe Point 推进。从 TiDB 6.1 版本起，计算 Safe Point 时会考虑内部事务的 startTS，从而解决内部事务因访问数据被清理掉而导致失败的问题。带来的负面影响是如果内部事务运行时间过长，会导致 Safe Point 长时间不推进，进而会影响业务性能。
```
Comment (Oreoxmt) [inline garbage-collection-configuration.md:36]:
```suggestion
在 TiDB 6.1.0 之前的版本中，TiDB 内部事务不会影响 GC Safe Point 推进。从 TiDB 6.1.0 版本起，计算 Safe Point 时会考虑内部事务的 startTS，从而解决内部事务因访问数据被清理掉而导致失败的问题。带来的负面影响是如果内部事务运行时间过长，会导致 Safe Point 长时间不推进，进而会影响业务性能。
```

File: garbage-collection-configuration.md [hunk 3]
```diff
@@ -31,6 +31,12 @@ TiDB 5.0 及之后的版本不再需要向各个 TiKV Region 都发送触发 GC
...
+
+ TiDB 6.1 版本 引入了配置变量 [`tidb_gc_max_wait_time`](#tidb_gc_max_wait_time-从-v610-版本开始引入) 控制活跃事务阻塞 GC Safe Point 推进的最长时间，超过该值后 GC Safe Point会强制向后推进。
```
Comment (Oreoxmt) [inline garbage-collection-configuration.md:38]:
```suggestion
 TiDB v6.1.0 引入了配置变量 [`tidb_gc_max_wait_time`](#tidb_gc_max_wait_time-从-v610-版本开始引入) 控制活跃事务阻塞 GC Safe Point 推进的最长时间，超过该值后 GC Safe Point 会强制向后推进。
```

File: garbage-collection-configuration.md [hunk 4]
```diff
@@ -31,6 +32,12 @@ TiDB 5.0 及之后的版本不再需要向各个 TiKV Region 都发送触发 GC
 
 如果要了解 TiDB 历史版本中 GC 配置的变化信息，请使用左侧导航栏中的 _"TIDB 版本选择器"_ 切换到本文档的历史版本。
 
+## TiDB 6.1.0 引入的变化
+
+在 TiDB 6.1.0 之前的版本中，TiDB 内部事务不会影响 GC safe point 推进。从 TiDB 6.1.0 版本起，计算 safe point 时会考虑内部事务的 startTS，从而解决内部事务因访问数据被清理掉而导致失败的问题。带来的负面影响是如果内部事务运行时间过长，会导致 safe point 长时间不推进，进而会影响业务性能。
+
+ TiDB v6.1.0 引入了系统变量 [`tidb_gc_max_wait_time`](/system-variables.md#tidb_gc_max_wait_time-从-v610-版本开始引入) 控制活跃事务阻塞 GC safe point 推进的最长时间，超过该值后 GC safe point 会强制向后推进。
```
Comment (Oreoxmt) [inline garbage-collection-configuration.md:39]:
```suggestion
TiDB v6.1.0 引入了系统变量 [`tidb_gc_max_wait_time`](/system-variables.md#tidb_gc_max_wait_time-从-v610-版本开始引入) 控制活跃事务阻塞 GC safe point 推进的最长时间，超过该值后 GC safe point 会强制向后推进。
```

**Review by cfzjywxk — APPROVED** [review]

File: garbage-collection-configuration.md [hunk 5]
```diff
@@ -31,6 +32,12 @@ TiDB 5.0 及之后的版本不再需要向各个 TiKV Region 都发送触发 GC
 
 如果要了解 TiDB 历史版本中 GC 配置的变化信息，请使用左侧导航栏中的 _"TIDB 版本选择器"_ 切换到本文档的历史版本。
 
+## TiDB 6.1.0 引入的变化
+
+在 TiDB 6.1.0 之前的版本中，TiDB 内部事务不会影响 GC safe point 推进。从 TiDB 6.1.0 版本起，计算 safe point 时会考虑内部事务的 startTS，从而解决内部事务因访问数据被清理掉而导致失败的问题。带来的负面影响是如果内部事务运行时间过长，会导致 safe point 长时间不推进，进而会影响业务性能。
```
Comment (TomShawn) [inline garbage-collection-configuration.md:37]:
```suggestion
在 TiDB 6.1.0 之前的版本中，TiDB 内部事务不会影响 GC safe point 推进。从 TiDB 6.1.0 版本起，计算 safe point 时会考虑内部事务的 startTS，从而解决内部事务因访问的数据被清理掉而导致失败的问题。带来的负面影响是如果内部事务运行时间过长，会导致 safe point 长时间不推进，进而会影响业务性能。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**